crates/services/txpool_v2/src/tests/universe.rs

```diff
@@ -93,6 +94,7 @@ use super::mocks::{
 // the byte and gas price fees.
 pub const TEST_COIN_AMOUNT: u64 = 100_000_000u64;
 pub const GAS_LIMIT: Word = 100000;
+pub const DEFAULT_EXPIRATION_HEIGHT: BlockHeight = BlockHeight::new(1000);
```

Comment:
```suggestion
pub const DEFAULT_EXPIRATION_HEIGHT: BlockHeight = BlockHeight::new(10);
```
1_000 seems like state would change too much, no?

Reply:
This is a test onyl feature to be sure it doesn't change the existing tests that may be running for long. In the real world the default value is "no expiration"

Reply:
should the default value be "no expiration"? is that for backwards compatibility or some other reason?

Reply:
For tests I find it easier to write test in the future then and so they all shared a same constant instead of magical numbers in each tests that would endup in a constant

Reply:
oh, I'm curious about the "real world" default for tx expiry, not the tests here :)

Reply:
In real world it's "no expiration"